build.sbt

```diff
@@ -513,3 +513,8 @@ def initCommands(additionalImports: String*) =
 // Everything is driven through release steps and the http4s* variables
 // This won't actually release unless on Travis.
 addCommandAlias("ci", ";clean ;release with-defaults")
+
+// VSCode Test Helper
+// Compile Both Test and Compile Scope
+// https://github.com/sbt/sbt/issues/4007
+addCommandAlias("compile", "test:compile")
```

Comment:
Hmm, yeah.

Reply:
Can you add global command aliases?  I like to make editor quirks global, because my editor is global.  This seems like it would scratch your itch and not make others itch, if it works.

Reply:
I'll give it a shot.